CRM/Contact/BAO/GroupContact.php

```diff
@@ -9,59 +9,63 @@
  +--------------------------------------------------------------------+
  */
 
+use Civi\Api4\SubscriptionHistory;
+use Civi\Core\Event\PostEvent;
+
 /**
  *
  * @package CRM
  * @copyright CiviCRM LLC https://civicrm.org/licensing
  */
-class CRM_Contact_BAO_GroupContact extends CRM_Contact_DAO_GroupContact {
+class CRM_Contact_BAO_GroupContact extends CRM_Contact_DAO_GroupContact implements \Civi\Test\HookInterface {
```

Comment:
```suggestion
class CRM_Contact_BAO_GroupContact extends CRM_Contact_DAO_GroupContact implements \Civi\Core\HookInterface {
```

Reply:
@eileenmcnaughton this was renamed since Jan

Reply:
@colemanw OK - fixed